05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,673 @@
+
+#### The syntax {-}
+
+To specify the Jackknife method, the type would be `JK1`, `JKn`, or `JK2`. Additionally, the overall multiplier for JK1 is specified with the scale argument, whereas the replicate specific multiplier ($\alpha_r) is specified with the rscales argument.
+
+Consider a case for the JK1 method where the multiplier, $(R-1)/R=19/20=0.95$ and the dataset had WT0 for the main weight and had 20 JK1 weights indicated WT1, WT2, ..., WT20, then the syntax would be
+
+```r
+des_jk1 <- dat %>%
+  as_survey_rep(weights = WT0, repweights= num_range("WT", 1:20),
+                type="JK1", mse=TRUE, scale=0.95)
+```
+
+Consider a case for the JKn method where $\alpha_r=0.1$ for all replicates and the dataset had WT0 for the main weight and had 20 JK1 weights indicated WT1, WT2, ..., WT20, then the syntax would be:
+
+```r
+des_jkn <- dat %>%
+  as_survey_rep(weights = WT0, repweights= num_range("WT", 1:20),
+                type="JKN", mse=TRUE, rscales=rep(0.1, 20))
+```
+
+#### Example {-}
+
+The American Community Survey releases public use microdata with JK1 weights at the person and household level. This example includes data at the household level where the replicate weights are specified as WGTP1, ..., WGTP80 and the main weight is WGTP^[American Community Survey 2016-2020 5-Year PUMS File: https://www2.census.gov/programs-surveys/acs/tech_docs/pums/ACS2016_2020_PUMS_README.pdf]. Using the {tidycensus} package^[tidycensus package: https://walker-data.com/tidycensus/], data is downloaded from the Census API. This request gets data for each person in each household in two Public Use Microdata Areas (PUMAs) in Durham County, NC^[Public Use Microdata Areas in North Carolina: https://www.census.gov/geographies/reference-maps/2010/geo/2010-pumas/north-carolina.html]. The variables requested are NP (number of persons in household), BDSP (number of bedrooms),  HINCP (household income), and TYPEHUGQ (type of household). By default, several other variables will come along including SERIALNO (a unique identifier for each household), SPORDER (a unique identifier for each person within each household), PUMA, ST (state), person weight (PWGTP), and the household weights (WGTP, WGTP1, ..., WGTP80). Filtering to records where SPORDER=1 yields only one record per household and TYPEHUGQ=1 filters to only households and not group quarters.
+
+```{r}
+#| label: acsexamp
+#| cache: TRUE
+#| results: 'hide'
+library(tidycensus)
+
+pums_in <- get_pums(variables=c("NP", "BDSP", "HINCP"), state="37", 
+                    puma=c("01301", "01302"), rep_weights = "housing", 
+                    year=2020, survey="acs5",
+                    variables_filter=list(SPORDER=1, TYPEHUGQ=1))
+```
+
+```{r}
+#| label: acsexampcont
+#| cache: TRUE
+#| dependson: 'acsexamp'
+pums_in
+
+des_acs <- pums_in %>%
+    as_survey_rep(weights = WGTP, repweights= num_range("WGTP", 1:80),
+                type="JK1", mse=TRUE, scale=4/80)
+
+des_acs
+
+summary(des_acs)
```

Comment:
This output goes off the page on the rendered PDF, do we have anyway of fixing that?